catanatron/models/player.py

```diff
@@ -42,6 +42,9 @@ def has_knight_card(self):
     def has_year_of_plenty_card(self):
         return self.development_deck.count(DevelopmentCard.YEAR_OF_PLENTY) > 0
 
+    def has_monopoly_card(self):
```

Comment:
I think eventually we can make a more generic (has_dev_card(card_type)) method to replace these